turingschool-examples/ideabox-boilerplate
# Refactor

Is this a fix or a feature?
Feature

What is the change/fix?
- Refactored JS to be DRYer
- Refactored CSS for Lighthouse audit

Where should the reviewer start?
- Top of JS file

How should this be tested?
- Open index.html in browser and check functionality

Screenshots(if appropriate)
![image](https://user-images.githubusercontent.com/13261139/98171781-85ecdb80-1ead-11eb-8839-04a763d696b6.png)
